src/agent/src/sandbox.rs

```diff
@@ -233,14 +233,29 @@ impl Sandbox {
             online_memory(&self.logger)?;
         }
 
-        let cpuset = rustjail_cgroups::fs::get_guest_cpuset()?;
+        let guest_cpuset = rustjail_cgroups::fs::get_guest_cpuset()?;
 
         for (_, ctr) in self.containers.iter() {
+            let cpu = ctr
```

Comment:
please try to avoid `unwrap`, there are other alternatives like `?`

Reply:
`?` on `Option` does not match the fn's return type `Result`. To make it works, `std::option::NoneError` should convert to ``std::error::Error`.